php-practice/section8.php

```diff
@@ -128,3 +139,19 @@ function printGender($gender) {
  *
  * Then create a function that takes a title and echos out a witty comment about that title.
  */
+
+define('Fellowship of the Ring', 1);
+define('Two Towers', 2);
+define('Return of the King', 3);
+
+function printComment($title) {
+	if($title === 'Fellowship of the Ring') {
+		echo 'An incredible start to a trilogy!'.PHP_EOL;
+	} elseif ($title === 'Two Towers') {
+		echo 'Wow! Makes you want to read the Final Book!'.PHP_EOL;
+	} else {
+		echo 'I can\'t believe it\'s over!'.PHP_EOL;
+	}
+}
```

Comment:
That makes sense. I got confused by the quotes but I looked back over the instructions and that makes sense.